### ISSUE TITLE
Crash from ReadingTagTask

### ISSUE THREAD

**Issue by micolous** [micolous/metrodroid#521]
Android 9, OnePlus5, Metrodroid v2.9.42:

```
android.view.WindowManager$BadTokenException: 
  at android.view.ViewRootImpl.setView (ViewRootImpl.java:1022)
  at android.view.WindowManagerGlobal.addView (WindowManagerGlobal.java:368)
  at android.view.WindowManagerImpl.addView (WindowManagerImpl.java:94)
  at android.app.Dialog.show (Dialog.java:329)
  at android.app.AlertDialog$Builder.show (AlertDialog.java:1125)
  at au.id.micolous.metrodroid.activity.ReadingTagTask.onPostExecute (ReadingTagTask.kt:87)
  at au.id.micolous.metrodroid.activity.ReadingTagTask$start$1$1.run (ReadingTagTask.kt:128)
  at android.os.Handler.handleCallback (Handler.java:873)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6940)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:537)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
```

**Comment by micolous** [comment]
Looks like it's also in v2.9.40, multiple devices with Android 8 and 9:

```
android.view.WindowManager$BadTokenException: 
  at android.view.ViewRootImpl.setView (ViewRootImpl.java:798)
  at android.view.WindowManagerGlobal.addView (WindowManagerGlobal.java:356)
  at android.view.WindowManagerImpl.addView (WindowManagerImpl.java:93)
  at android.app.Dialog.show (Dialog.java:329)
  at android.app.AlertDialog$Builder.show (AlertDialog.java:1125)
  at au.id.micolous.metrodroid.activity.ReadingTagTask.onPostExecute (ReadingTagTask.kt:87)
  at au.id.micolous.metrodroid.activity.ReadingTagTask$start$1$1.run (ReadingTagTask.kt:128)
  at android.os.Handler.handleCallback (Handler.java:873)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6718)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
```

**Comment by micolous** [comment]
Fixed by #527
